### DISCUSSION TITLE
Add automatically disable debuggable

### DISCUSSION THREAD

**Discussion by binx6** [AndroidIDEOfficial/AndroidIDE#1187] [Ideas]
The IDE always enables`android:debuggable="true"`,by default when packaging an app. I know this is to make it easier to debug the app, but it's not something most people need very often. And apps built with the IDE always open with an annoying **Failed to bind to AndroidIDE** Toast, hoping to add an option to disable them with one click so I don't have to manually modify them.

**Comment by itsaky** [comment]
The `android:debuggable="true"` attribute is added by the Android Gradle Plugin, not AndroidIDE. The attribute is added only for debug builds.

The `Failed to bind to AndroidIDE` message you see when the application is opened is because the logsender service is trying to bind to the log receiver in AndroidIDE, but AndroidIDE is not in the background.

You can build and install a release build of your application to fix both of these.

**Reply by binx6** [reply]
I see
